stratum1.yml

```diff
@@ -0,0 +1,17 @@
+- name: CVMFS
+  hosts: cvmfsstratum1servers
+  vars:
+    #cvmfs_srv_device: /dev/sdb
+    galaxy_cvmfs_repos_enabled: true
```

Comment:
This still says `galaxy`?

Reply:
Yes, they still use galaxy_* variables in lots of the ansible files. For now I changed our vars file by renaming all our variables to eessi_*, and pointing the galaxy_* to the corresponding eessi_* variables in that file. Now we can use eessi_* in our playbooks, which makes it a bit cleaner.